src/SurfaceFluxes.jl

```diff
@@ -353,12 +356,64 @@ obukhov_length(sfc::SurfaceFluxConditions) = sfc.L_MO
 
 non_zero(v::FT) where {FT} = abs(v) < eps(FT) ? v + sqrt(eps(FT)) : v
 
+
+function compute_richardson_number(sc::AbstractSurfaceConditions{FT}, DSEᵥ_in, DSEᵥ_sfc, grav) where {FT}
```

Comment:
```suggestion
function compute_richardson_number(sc::AbstractSurfaceConditions, DSEᵥ_in, DSEᵥ_sfc, grav)
```
we don't need to specialize on `FT` here if it's not being used